components/prism-log.js

```diff
@@ -66,7 +70,7 @@ Prism.languages.log = {
 	},
 
 	'uuid': {
-		pattern: /\b\w{8}-\w{4}-\w{4}-\w{4}-\w{12}\b/,
+		pattern: /\b\w{8}-?\w{4}-?\w{4}-?\w{4}-?\w{12}\b/,
```

Comment:
This is something I did wrong, but it will really be a problem now: we really need to enforce hexadecimal digits.

```suggestion
		pattern: /\b[0-9a-f]{8}-?[0-9a-f]{4}-?[0-9a-f]{4}-?[0-9a-f]{4}-?[0-9a-f]{12}\b/i,
```

Reply:
Yeah, I noticed this rule was catching random groups of 32 alphanumeric characters now. I'll switch it to hexadecimal.